portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,58 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+html, body {
+  height: 100%; 
 }
 
-#greeting-container {
-  margin-top: 20px;
+.wrapper {  
+  display: flex;
+  justify-content: center;
+  margin: auto;
+  max-width: 800px;
 }
+
+.welcome {
+  height: 100%;
+}
+
+.left {
```

Comment:
Since these classes (left/right) don't really do any positioning, it may be easier to give them names that refer to what they are instead (like profile pic and summary, something like that).